test/test_sparse_csr.py

```diff
@@ -1990,25 +2020,34 @@ def run_test(shape, nnz, index_type):
     @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16))
     def test_transpose(self, device, dtype):
 
-        def run_test(shape, nnz, index_type, dim0, dim1):
+        def run_test(shape, nnz, index_type, dim0, dim1, transposed):
             a = self.genSparseCSRTensor(shape, nnz, dtype=dtype, device=device, index_dtype=index_dtype)
+            if transposed:
+                a = a.transpose(0, 1)
+                self.assertEqual(a.layout, torch.sparse_csc)
```

Comment:
Do we have a test for checking that `transpose` of a sparse compressed tensor is a view operation?

Reply:
I added one based on the information in test_view_ops